### ISSUE TITLE
Tonal Parses 7(something) chords incorrectly

### ISSUE THREAD

**Issue by roelofvkr** [danigb/tonal#20]
I'm using tonal in combination with Midiguchi, and one thing that is happening for me right now, which i think is an issue with the library.
I'm using `tonal.chord.detect([notes])` to get the chord name and then later on i use `tonal.chord.parse([chordname that i got from detect])` to get the chord type. 
This all works fine, but one issues i'm having right now is that whenever `tonal.chord.notes()` encounters a 7something chord (7sus4, 7sus2, 7add6) it won't return notes (it'll just return an empty array) and when i try to parse it i get back `{ type: 'add6', tonic: 'Eb7' }` (if the chord was a "Eb7add6").
Is there anything i can do to fix this?
Thanks

**Comment by danigb** [comment]
Hi,

I'm sorry for the inconvenience. It would help if you write a little piece of code that I can use to reproduce your issue.

**Comment by roelofvkr** [comment]
Yeah of course, here's a test function i just wrote to see if i could get it to work properly, it's very basic but it  still gives the same problem
```Javascript
function test () {
  console.log(tonal.chord.parse("Eb7add6"))
  console.log(tonal.chord.notes("Eb7add6"));
}
```

when I call that function i get this result:
```
{ type: 'add6', tonic: 'Eb7' }
[]
```

As you can see the tonic returned is a chord and the notes array returns empty.

**Comment by danigb** [comment]
Hi @roelofvkr 

I've fixed the issue as you can see [here](https://github.com/danigb/tonal/blob/master/packages/chord/test/chord.test.js#L30) and [here](https://github.com/danigb/tonal/blob/master/packages/chord/test/chord.test.js#L41). I've published a patch version with the fixes in npm (0.69.1). Hope it helps. Please reopen the issue if the problem persists. Thanks for reporting.
